grails-bootstrap/src/main/groovy/grails/io/IOUtils.groovy

```diff
@@ -269,6 +269,11 @@ class IOUtils extends SpringIOUtils {
                 def buildClassespath = BuildSettings.BUILD_CLASSES_PATH.replace('/', File.separator)
                 if(rootPath.contains(buildClassespath)) {
```

Comment:
For multi project builds executed via `Application.groovy`, this was always returning false due to the compiled path being different than the `BUILD_CLASSES_PATH`